diff -ur cmake-3.11.2.orig/Source/CMakeLists.txt cmake-3.11.2/Source/CMakeLists.txt
--- cmake-3.11.2.orig/Source/CMakeLists.txt	2018-05-29 14:54:25.522904245 -0400
+++ cmake-3.11.2/Source/CMakeLists.txt	2018-05-29 14:57:09.553947612 -0400
@@ -949,8 +949,8 @@
 endif()
 if(WIN32 OR (CYGWIN AND LibUUID_FOUND))
   set(CPACK_SRCS ${CPACK_SRCS}
-    CPack/Wix/cmCMakeToWixPath.cxx
-    CPack/Wix/cmCMakeToWixPath.h
+    CPack/WiX/cmCMakeToWixPath.cxx
+    CPack/WiX/cmCMakeToWixPath.h
     CPack/WiX/cmCPackWIXGenerator.cxx
     CPack/WiX/cmCPackWIXGenerator.h
     CPack/WiX/cmWIXAccessControlList.cxx
