diff -ur a/doc/interpreter/Makefile.in b/doc/interpreter/Makefile.in
--- a/doc/interpreter/Makefile.in	2013-12-27 17:01:52.000000000 -0500
+++ b/doc/interpreter/Makefile.in	2014-01-21 12:47:59.618865568 -0500
@@ -2435,8 +2435,10 @@
 	  if test -d "$$d2"; then \
 	    echo " $(MKDIR_P) '$(DESTDIR)$(htmldir)/$$f'"; \
 	    $(MKDIR_P) "$(DESTDIR)$(htmldir)/$$f" || exit 1; \
-	    echo " $(INSTALL_DATA) '$$d2'/* '$(DESTDIR)$(htmldir)/$$f'"; \
-	    $(INSTALL_DATA) "$$d2"/* "$(DESTDIR)$(htmldir)/$$f" || exit $$?; \
+	    for htm in "$$d2"/*; do \
+	      echo " $(INSTALL_DATA) '$$htm' '$(DESTDIR)$(htmldir)/$$f'"; \
+	      $(INSTALL_DATA) "$$htm" "$(DESTDIR)$(htmldir)/$$f" || exit $$?; \
+	    done; \
 	  else \
 	    list2="$$list2 $$d2"; \
 	  fi; \
