diff -u fontconfig-2.11.0.orig/Makefile.in fontconfig-2.11.0/Makefile.in
--- fontconfig-2.11.0.orig/Makefile.in	2014-02-01 14:30:40.000000000 -0500
+++ fontconfig-2.11.0/Makefile.in	2014-02-01 14:30:58.000000000 -0500
@@ -997,7 +997,7 @@
 	mv $@.tmp $@
 
 install-data-local: fonts.conf
-	$(mkinstalldirs) $(DESTDIR)$(baseconfigdir) $(DESTDIR)$(fc_cachedir)
+	$(mkinstalldirs) $(DESTDIR)$(baseconfigdir) #$(DESTDIR)$(fc_cachedir)
 	if [ -f $(DESTDIR)$(baseconfigdir)/fonts.conf ]; then \
 	  echo "backing up existing $(DESTDIR)$(baseconfigdir)/fonts.conf"; \
 	  mv $(DESTDIR)$(baseconfigdir)/fonts.conf $(DESTDIR)$(baseconfigdir)/fonts.conf.bak; \
