diff -ur glpk-4.65.orig/configure.ac glpk-4.65/configure.ac
--- glpk-4.65.orig/configure.ac	2019-02-20 14:31:36.960931258 -0500
+++ glpk-4.65/configure.ac	2019-02-20 14:36:12.463581439 -0500
@@ -190,6 +190,14 @@
 esac
 AC_SUBST([NOUNDEFINED])
 
+AC_CHECK_LIB([z], [gzopen])
+
+AC_CHECK_LIB([amd], [amd_1])
+AC_CHECK_LIB([colamd], [colamd])
+
+#CPPFLAGS="$CPPFLAGS -I/usr/include/suitesparse"
+AC_CHECK_HEADER([suitesparse/amd.h])
+
 AC_CONFIG_FILES(
    [src/Makefile examples/Makefile Makefile])
 AC_OUTPUT
diff -ur glpk-4.65.orig/src/Makefile.am glpk-4.65/src/Makefile.am
--- glpk-4.65.orig/src/Makefile.am	2019-02-20 14:31:36.988931934 -0500
+++ glpk-4.65/src/Makefile.am	2019-02-20 14:37:43.921787648 -0500
@@ -6,10 +6,8 @@
 
 libglpk_la_CPPFLAGS = \
 -I$(srcdir) \
--I$(srcdir)/amd \
 -I$(srcdir)/api \
 -I$(srcdir)/bflib \
--I$(srcdir)/colamd \
 -I$(srcdir)/draft \
 -I$(srcdir)/env \
 -I$(srcdir)/intopt \
@@ -18,8 +16,7 @@
 -I$(srcdir)/mpl \
 -I$(srcdir)/npp \
 -I$(srcdir)/proxy \
--I$(srcdir)/simplex \
--I$(srcdir)/zlib
+-I$(srcdir)/simplex
 
 libglpk_la_LDFLAGS = \
 -version-info 43:0:3 \
@@ -27,18 +24,6 @@
 ${NOUNDEFINED}
 
 libglpk_la_SOURCES = \
-amd/amd_1.c \
-amd/amd_2.c \
-amd/amd_aat.c \
-amd/amd_control.c \
-amd/amd_defaults.c \
-amd/amd_dump.c \
-amd/amd_info.c \
-amd/amd_order.c \
-amd/amd_post_tree.c \
-amd/amd_postorder.c \
-amd/amd_preprocess.c \
-amd/amd_valid.c \
 api/advbas.c \
 api/asnhall.c \
 api/asnlp.c \
@@ -104,7 +89,6 @@
 bflib/scfint.c \
 bflib/sgf.c \
 bflib/sva.c \
-colamd/colamd.c \
 draft/bfd.c \
 draft/bfx.c \
 draft/glpapi06.c \
@@ -202,21 +186,6 @@
 simplex/spxprob.c \
 simplex/spychuzc.c \
 simplex/spychuzr.c \
-simplex/spydual.c \
-zlib/adler32.c \
-zlib/compress.c \
-zlib/crc32.c \
-zlib/deflate.c \
-zlib/gzclose.c \
-zlib/gzlib.c \
-zlib/gzread.c \
-zlib/gzwrite.c \
-zlib/inffast.c \
-zlib/inflate.c \
-zlib/inftrees.c \
-zlib/trees.c \
-zlib/uncompr.c \
-zlib/zio.c \
-zlib/zutil.c
+simplex/spydual.c
 
 ## eof ##
diff -ur glpk-4.65.orig/src/draft/glpmat.c glpk-4.65/src/draft/glpmat.c
--- glpk-4.65.orig/src/draft/glpmat.c	2019-02-20 15:21:37.117944413 -0500
+++ glpk-4.65/src/draft/glpmat.c	2019-02-20 15:22:58.407893375 -0500
@@ -25,8 +25,8 @@
 #include "env.h"
 #include "glpmat.h"
 #include "qmd.h"
-#include "amd.h"
-#include "colamd.h"
+#include <suitesparse/amd.h>
+#include <suitesparse/colamd.h>
 
 /*----------------------------------------------------------------------
 -- check_fvs - check sparse vector in full-vector storage format.
diff -ur glpk-4.65.orig/src/env/stdc.c glpk-4.65/src/env/stdc.c
--- glpk-4.65.orig/src/env/stdc.c	2019-02-20 15:29:18.821040444 -0500
+++ glpk-4.65/src/env/stdc.c	2019-02-20 15:40:06.948676811 -0500
@@ -49,7 +49,7 @@
 
 /* MS Windows version *************************************************/
 
-#elif defined(__WOE__)
+#elif defined(__WOE__) || defined(_WIN32)
 
 #include "stdc.h"
 
Only in glpk-4.65/src/env: .time.c.swp
