diff -ur gnuplot-4.6.4.orig/config/mingw/Makefile gnuplot-4.6.4/config/mingw/Makefile
--- gnuplot-4.6.4.orig/config/mingw/Makefile	2014-07-24 20:30:33 -0400
+++ gnuplot-4.6.4/config/mingw/Makefile	2014-07-24 20:30:50 -0400
@@ -382,8 +382,9 @@
 endif
 
 ifdef ICONV
-	CFLAGS += -DHAVE_ICONV
-	TERMLIBS += -liconv
+	CFLAGS += -DHAVE_ICONV $(ICONV_CFLAGS)
+        LDFLAGS += $(ICONV_LDFLAGS)
+	TERMLIBS += -liconv 
 endif
 
 ifdef GNU_RC
