diff --git "a/biosig.c" "b/biosig.c"
index 5872cef..ed7b64a 100644
--- "a/biosig.c"	
+++ "b/biosig.c"
@@ -53,7 +53,26 @@
 #include <locale.h>
 #include <math.h>      // define macro isnan()
 #include <stdlib.h>
+#ifdef _WIN32
+// Can't include sys/stat.h or sopen is declared twice.
+#include <sys/types.h>
+struct stat {
+  _dev_t st_dev;
+  _ino_t st_ino;
+  unsigned short st_mode;
+  short st_nlink;
+  short st_uid;
+  short st_gid;
+  _dev_t st_rdev;
+  _off_t st_size;
+  time_t st_atime;
+  time_t st_mtime;
+  time_t st_ctime;
+};
+int __cdecl stat(const char *_Filename,struct stat *_Stat);
+#else
 #include <sys/stat.h>
+#endif
 
 
 #ifdef WITH_CURL

diff --git "a/t210/sopen_heka_read.c" "b/t210/sopen_heka_read.c"
index d256a15..6025211 100644
--- "a/t210/sopen_heka_read.c"	
+++ "b/t210/sopen_heka_read.c"
@@ -24,7 +24,26 @@
 #include <ctype.h>
 #include <stdlib.h>
 #include <string.h>
+#ifdef _WIN32
+// Can't include sys/stat.h or sopen is declared twice.
+#include <sys/types.h>
+struct stat {
+  _dev_t st_dev;
+  _ino_t st_ino;
+  unsigned short st_mode;
+  short st_nlink;
+  short st_uid;
+  short st_gid;
+  _dev_t st_rdev;
+  _off_t st_size;
+  time_t st_atime;
+  time_t st_mtime;
+  time_t st_ctime;
+};
+int __cdecl stat(const char *_Filename,struct stat *_Stat);
+#else
 #include <sys/stat.h>
+#endif
 
 #include "../biosig.h"
 

diff --git "a/test0/sandbox.c" "b/test0/sandbox.c"
index eed47b7..309c009 100644
--- "a/test0/sandbox.c"	
+++ "b/test0/sandbox.c"
@@ -26,7 +26,26 @@
 #include <ctype.h>
 #include <stdlib.h>
 #include <string.h>
+#ifdef _WIN32
+// Can't include sys/stat.h or sopen is declared twice.
+#include <sys/types.h>
+struct stat {
+  _dev_t st_dev;
+  _ino_t st_ino;
+  unsigned short st_mode;
+  short st_nlink;
+  short st_uid;
+  short st_gid;
+  _dev_t st_rdev;
+  _off_t st_size;
+  time_t st_atime;
+  time_t st_mtime;
+  time_t st_ctime;
+};
+int __cdecl stat(const char *_Filename,struct stat *_Stat);
+#else
 #include <sys/stat.h>
+#endif
 
 #include "../biosig.h"
 

