diff -ur biosig4c++-1.9.4.orig/Makefile.in biosig4c++-1.9.4/Makefile.in
--- biosig4c++-1.9.4.orig/Makefile.in	2019-05-06 15:20:47.958318279 -0400
+++ biosig4c++-1.9.4/Makefile.in	2019-05-06 15:21:45.810270194 -0400
@@ -241,7 +241,7 @@
 	  LDLIBS     += -lsuitesparseconfig
 	endif
 endif
-ifeq (1,@HAVE_DCMTK@)
+ifeq (0,@HAVE_DCMTK@)
 	DEFINES      += -D=WITH_DCMTK
 	LDLIBS       += -ldcmdata -loflog -lofstd
 	SOURCES      += t210/sopen_dcmtk_read.cpp
