diff -ur libgeotiff-1.4.0.orig/libxtiff/Makefile.am libgeotiff-1.4.0/libxtiff/Makefile.am
--- libgeotiff-1.4.0.orig/libxtiff/Makefile.am	2014-06-28 09:56:02.000000000 -0400
+++ libgeotiff-1.4.0/libxtiff/Makefile.am	2014-06-28 11:57:04.000000000 -0400
@@ -7,5 +7,5 @@
 AM_CFLAGS = -I../ $(TIFF_CFLAGS) 
 
 libxtiff_la_SOURCES = xtiff.c
-
+libxtiff_la_LIBADD = -ltiff
 noinst_LTLIBRARIES = libxtiff.la
diff -ur libgeotiff-1.4.0.orig/Makefile.am libgeotiff-1.4.0/Makefile.am
--- libgeotiff-1.4.0.orig/Makefile.am	2014-06-28 09:56:02.000000000 -0400
+++ libgeotiff-1.4.0/Makefile.am	2014-06-28 09:56:53.000000000 -0400
@@ -108,7 +108,7 @@
                              csv/unit_of_measure.c 
 endif
 
-libgeotiff_la_LDFLAGS = -version-info 3:0:1
+libgeotiff_la_LDFLAGS = -version-info 3:0:1 -no-undefined
 
 libgeotiff_la_LIBADD = libxtiff/libxtiff.la
 
diff -ur libgeotiff-1.4.0.orig/Makefile.in libgeotiff-1.4.0/Makefile.in
--- libgeotiff-1.4.0.orig/Makefile.in	2014-06-28 09:56:02.000000000 -0400
+++ libgeotiff-1.4.0/Makefile.in	2014-06-28 17:37:48.000000000 -0400
@@ -448,7 +448,7 @@
 	geo_get.c geo_names.c geo_new.c geo_normalize.c geo_print.c \
 	geo_set.c geo_simpletags.c geo_tiffp.c geo_trans.c geo_write.c \
 	geo_strtod.c geotiff_proj4.c $(am__append_1)
-libgeotiff_la_LDFLAGS = -version-info 3:0:1
+libgeotiff_la_LDFLAGS = -version-info 3:0:1 -no-undefined
 libgeotiff_la_LIBADD = libxtiff/libxtiff.la
 lib_LTLIBRARIES = libgeotiff.la
 EXTRA_DIST = makefile.vc \
diff -burN libgeotiff-1.4.0.orig/configure.ac libgeotiff-1.4.0/configure.ac
--- libgeotiff-1.4.0.orig/configure.ac	2015-06-21 03:50:03.341076086 +0200
+++ libgeotiff-1.4.0/configure.ac	2015-06-21 03:50:39.741125191 +0200
@@ -280,7 +280,7 @@
   PROJ_INC=-I$PROJ_INC
 
   AC_SUBST(PROJ_INC,$PROJ_INC)
-  AC_DEFINE(HAVE_PROJECTS_H)
+  AC_DEFINE([HAVE_PROJECTS_H], [], [Description])
   AC_DEFINE(HAVE_LIBPROJ)
   AC_SUBST([HAVE_LIBPROJ])
   PROJ_CONFIG="yes"
