diff -uNr a/scons/gallium.py b/scons/gallium.py
--- a/scons/gallium.py	2017-09-17 19:03:22.000000000 -0400
+++ b/scons/gallium.py	2017-10-04 19:24:29.499373623 -0400
@@ -580,9 +580,6 @@
                 '-Wl,--dynamicbase', # ASLR
             ]
             # Avoid depending on gcc runtime DLLs
-            linkflags += ['-static-libgcc']
-            if 'w64' in env['CC'].split('-'):
-                linkflags += ['-static-libstdc++']
             # Handle the @xx symbol munging of DLL exports
             shlinkflags += ['-Wl,--enable-stdcall-fixup']
             #shlinkflags += ['-Wl,--kill-at']
diff -uNr a/scons/llvm.py b/scons/llvm.py
--- a/scons/llvm.py	2018-12-27 17:02:01.000000000 +0100
+++ b/scons/llvm.py	2019-01-21 18:47:28.456846140 +0100
@@ -122,10 +122,7 @@
                 'LLVMIRReader', 'LLVMAsmParser',
                 'LLVMDemangle', 'LLVMGlobalISel', 'LLVMDebugInfoMSF',
                 'LLVMBinaryFormat',
+                'version', 'uuid', 'ole32', 'oleaut32', 'z',
             ])
-            if env['platform'] == 'windows' and env['crosscompile']:
-                # LLVM 5.0 requires MinGW w/ pthreads due to use of std::thread and friends.
-                assert env['gcc']
-                env['CXX'] = env['CXX'] + '-posix'
         elif llvm_version >= distutils.version.LooseVersion('4.0'):
             env.Prepend(LIBS = [

