diff -ur mesa-20.0.6.orig/scons/gallium.py mesa-20.0.6/scons/gallium.py
--- mesa-20.0.6.orig/scons/gallium.py	2020-05-12 07:11:05.849629292 -0400
+++ mesa-20.0.6/scons/gallium.py	2020-05-12 07:12:17.601633140 -0400
@@ -617,9 +617,6 @@
                 '-Wl,--dynamicbase', # ASLR
             ]
             # Avoid depending on gcc runtime DLLs
-            linkflags += ['-static-libgcc']
-            if 'w64' in env['CC'].split('-'):
-                linkflags += ['-static-libstdc++']
             # Handle the @xx symbol munging of DLL exports
             shlinkflags += ['-Wl,--enable-stdcall-fixup']
             #shlinkflags += ['-Wl,--kill-at']
diff -ur mesa-20.0.6.orig/scons/llvm.py mesa-20.0.6/scons/llvm.py
--- mesa-20.0.6.orig/scons/llvm.py	2020-05-12 07:11:05.849629292 -0400
+++ mesa-20.0.6/scons/llvm.py	2020-05-12 07:49:13.605280341 -0400
@@ -243,7 +243,7 @@
         ])
 
         # Mingw-w64 zlib is required when building with LLVM support in MSYS2 environment
-        if host_platform.system().lower().startswith('mingw'):
+        if host_platform.system().lower().startswith('mingw') or env['crosscompile']:
             env.Append(LIBS = [
                  'z',
             ])
