--- a/fig2dev/fig2dev.h	2014-02-07 14:24:38.387076292 -0500
+++ b/fig2dev/fig2dev.h	2014-02-07 14:23:19.785715605 -0500
@@ -22,6 +22,7 @@
 #include <stdarg.h>
 #include "patchlevel.h"
 #include <math.h>
+#define NO_OLDNAMES
 #include <sys/file.h>
 #include <signal.h>
 #include <string.h>
--- a/fig2dev/dev/genemf.h	2010-03-16 14:18:59.000000000 -0400
+++ b/fig2dev/dev/genemf.h	2014-02-07 14:21:12.785972417 -0500
@@ -62,6 +62,8 @@
 # endif
 #endif
 
+#define BYTE_ORDER 1234
+
 #if defined(BYTE_ORDER) && (BYTE_ORDER == LITTLE_ENDIAN)
 
 # define htofs(x) (x)
