diff -ur fontconfig-2.10.93-orig/Makefile.in fontconfig-2.10.93/Makefile.in
--- fontconfig-2.10.93-orig/Makefile.in	2013-05-20 04:44:34 -0400
+++ fontconfig-2.10.93/Makefile.in	2013-11-08 13:54:43 -0500
@@ -977,7 +977,7 @@
 	mv $@.tmp $@
 
 install-data-local: fonts.conf
-	$(mkinstalldirs) $(DESTDIR)$(baseconfigdir) $(DESTDIR)$(fc_cachedir)
+	$(mkinstalldirs) $(DESTDIR)$(baseconfigdir) #$(DESTDIR)$(fc_cachedir)
 	if [ -f $(DESTDIR)$(baseconfigdir)/fonts.conf ]; then \
 	  echo "backing up existing $(DESTDIR)$(baseconfigdir)/fonts.conf"; \
 	  mv $(DESTDIR)$(baseconfigdir)/fonts.conf $(DESTDIR)$(baseconfigdir)/fonts.conf.bak; \
