--- a/glib/gatomic.c.orig	2013-07-29 12:05:27 -0400
+++ a/glib/gatomic.c	2013-07-29 12:07:46 -0400
@@ -475,6 +475,14 @@
 
 #if !defined (_MSC_VER) || _MSC_VER <= 1200
 #include "gmessages.h"
+
+#ifndef MemoryBarrier
+static inline void MemoryBarrier (void) {
+  long dummy = 0;
+  InterlockedExchange (&dummy, 1);
+}
+#endif
+
 /* Inlined versions for older compiler */
 static LONG
 _gInterlockedAnd (volatile guint *atomic,
