From http://lists.freedesktop.org/archives/gstreamer-commits/2014-February/077094.html

diff -ur libgcrypt-1.6.1.orig/random/rndhw.c libgcrypt-1.6.1/random/rndhw.c
--- libgcrypt-1.6.1.orig/random/rndhw.c	2014-08-16 22:16:09.991066266 -0400
+++ libgcrypt-1.6.1/random/rndhw.c	2014-08-16 22:19:19.025936274 -0400
@@ -69,7 +69,7 @@
   nbytes = 0;
   while (nbytes < 64)
     {
-#if defined(__x86_64__) && defined(__LP64__)
+#if defined(__x86_64__) // && defined(__LP64__)
       asm volatile
         ("movq %1, %%rdi\n\t"         /* Set buffer.  */
          "xorq %%rdx, %%rdx\n\t"      /* Request up to 8 bytes.  */
@@ -123,7 +123,7 @@
 #ifdef USE_DRNG
 # define RDRAND_RETRY_LOOPS	10
 # define RDRAND_INT	".byte 0x0f,0xc7,0xf0"
-# if defined(__x86_64__) && defined(__LP64__)
+# if defined(__x86_64__) //&& defined(__LP64__)
 #  define RDRAND_LONG	".byte 0x48,0x0f,0xc7,0xf0"
 # else
 #  define RDRAND_LONG	RDRAND_INT
