diff -u a/configure.ac b/configure.ac
--- a/configure.ac	2014-06-25 00:33:39.000000000 -0400
+++ b/configure.ac	2015-02-14 19:03:49.113882994 -0500
@@ -514,11 +514,17 @@
 AC_CHECK_FUNC([posix_memalign], [DEFINES="$DEFINES -DHAVE_POSIX_MEMALIGN"])
 
 dnl Check for pthreads
-AX_PTHREAD
+case "$host_os" in
+  mingw*)
+  ;;
+  *)
+    AX_PTHREAD
 dnl AX_PTHREADS leaves PTHREAD_LIBS empty for gcc and sets PTHREAD_CFLAGS
 dnl to -pthread, which causes problems if we need -lpthread to appear in
 dnl pkgconfig files.
-test -z "$PTHREAD_LIBS" && PTHREAD_LIBS="-lpthread"
+    test -z "$PTHREAD_LIBS" && PTHREAD_LIBS="-lpthread"
+  ;;
+esac
 
 dnl SELinux awareness.
 AC_ARG_ENABLE([selinux],
