diff -ur OpenBLAS-0.3.6.orig/driver/others/Makefile OpenBLAS-0.3.6/driver/others/Makefile
--- OpenBLAS-0.3.6.orig/driver/others/Makefile	2019-06-03 10:19:28.259909420 -0400
+++ OpenBLAS-0.3.6/driver/others/Makefile	2019-06-03 10:24:18.298986916 -0400
@@ -1,7 +1,7 @@
 TOPDIR	= ../..
 include ../../Makefile.system
 
-COMMONOBJS	 = memory.$(SUFFIX) xerbla.$(SUFFIX) c_abs.$(SUFFIX) z_abs.$(SUFFIX) openblas_set_num_threads.$(SUFFIX) openblas_get_num_threads.$(SUFFIX) openblas_get_num_procs.$(SUFFIX) openblas_get_config.$(SUFFIX) openblas_get_parallel.$(SUFFIX) openblas_error_handle.$(SUFFIX) openblas_env.$(SUFFIX)
+COMMONOBJS	 = memory.$(SUFFIX) c_abs.$(SUFFIX) z_abs.$(SUFFIX) openblas_set_num_threads.$(SUFFIX) openblas_get_num_threads.$(SUFFIX) openblas_get_num_procs.$(SUFFIX) openblas_get_config.$(SUFFIX) openblas_get_parallel.$(SUFFIX) openblas_error_handle.$(SUFFIX) openblas_env.$(SUFFIX)
 
 #COMMONOBJS	+= slamch.$(SUFFIX) slamc3.$(SUFFIX) dlamch.$(SUFFIX)  dlamc3.$(SUFFIX)
 
@@ -80,21 +80,18 @@
 
 ifeq ($(DYNAMIC_ARCH), 1)
 ifeq ($(ARCH),arm64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_arm64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_arm64.$(SUFFIX)
 else
 ifeq ($(ARCH),power)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_power.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_power.$(SUFFIX)
 else
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic.$(SUFFIX)
 endif
 endif
 else
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) parameter.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) parameter.$(SUFFIX)
 endif
 
-xerbla.$(SUFFIX) : xerbla.c
-	$(CC) $(CFLAGS) -c $< -o $(@F)
-
 dynamic.$(SUFFIX) : dynamic.c
 	$(CC) $(CFLAGS) -c $< -o $(@F)
 
@@ -182,9 +179,6 @@
 mulx.$(SUFFIX) : $(ARCH)/mulx.c
 	$(CC) $(CFLAGS) -c -DXDOUBLE -UCOMPLEX $< -o $(@F)
 
-xerbla.$(PSUFFIX) : xerbla.c
-	$(CC) $(PFLAGS) -c $< -o $(@F)
-
 parameter.$(PSUFFIX) : parameter.c ../../param.h
 	$(CC) $(PFLAGS) -c $< -o $(@F)
 
diff -ur OpenBLAS-0.3.6.orig/exports/gensymbol OpenBLAS-0.3.6/exports/gensymbol
--- OpenBLAS-0.3.6.orig/exports/gensymbol	2019-06-03 10:19:28.263909518 -0400
+++ OpenBLAS-0.3.6/exports/gensymbol	2019-06-03 10:24:49.179741367 -0400
@@ -38,7 +38,6 @@
     zhbmv,zhemm,zhemv,zher2,zher2k,zher,zherk,zhpmv,zhpr2,
     zhpr,zrotg,zscal,zswap,zsymm,zsyr2k,zsyrk,ztbmv,
     ztbsv,ztpmv,ztpsv,ztrmm,ztrmv,ztrsm,ztrsv,
-    xerbla,
     saxpby,daxpby,caxpby,zaxpby,
     sgeadd,dgeadd,cgeadd,zgeadd,
     somatcopy,
