diff -ur ghostscript-9.15.orig/base/gp_unifn.c ghostscript-9.15/base/gp_unifn.c
--- ghostscript-9.15.orig/base/gp_unifn.c	2015-02-16 07:47:31.050998971 -0500
+++ ghostscript-9.15/base/gp_unifn.c	2015-02-16 07:48:50.582467223 -0500
@@ -25,7 +25,11 @@
 
 /* Define the string to be concatenated with the file mode */
 /* for opening files without end-of-line conversion. */
+#if (defined(__MINGW32__) && __MINGW32__ == 1) || (defined(__CYGWIN__) && __CYGWIN__ == 1)
+const char gp_fmode_binary_suffix[] = "b";
+#else
 const char gp_fmode_binary_suffix[] = "";
+#endif
 
 /* Define the file modes for binary reading or writing. */
 #if (defined(__MINGW32__) && __MINGW32__ == 1) || (defined(__CYGWIN__) && __CYGWIN__ == 1)
