diff -ur fontconfig-2.11.0.orig/conf.d/Makefile.am fontconfig-2.11.0/conf.d/Makefile.am
--- fontconfig-2.11.0.orig/conf.d/Makefile.am	2014-02-22 20:53:48.000000000 -0500
+++ fontconfig-2.11.0/conf.d/Makefile.am	2014-02-22 20:59:45.000000000 -0500
@@ -88,9 +88,9 @@
 	@(echo cd $(DESTDIR)$(configdir);			\
 	  cd $(DESTDIR)$(configdir);				\
 	  for i in $(CONF_LINKS); do				\
-	    echo $(RM) $$i";" $(LN_S) $(DESTDIR)$(templatedir)/$$i .;	\
+	    echo $(RM) $$i";" cp $(DESTDIR)$(templatedir)/$$i .;	\
 	    $(RM) $$i;						\
-	    $(LN_S) $(DESTDIR)$(templatedir)/$$i .;		\
+	    cp $(DESTDIR)$(templatedir)/$$i .;		\
 	  done)
 uninstall-local:
 	@(echo cd $(DESTDIR)$(configdir);			\
diff -ur fontconfig-2.11.0.orig/conf.d/Makefile.in fontconfig-2.11.0/conf.d/Makefile.in
--- fontconfig-2.11.0.orig/conf.d/Makefile.in	2014-02-22 20:53:48.000000000 -0500
+++ fontconfig-2.11.0/conf.d/Makefile.in	2014-02-22 21:01:04.000000000 -0500
@@ -632,9 +632,9 @@
 	@(echo cd $(DESTDIR)$(configdir);			\
 	  cd $(DESTDIR)$(configdir);				\
 	  for i in $(CONF_LINKS); do				\
-	    echo $(RM) $$i";" $(LN_S) $(DESTDIR)$(templatedir)/$$i .;	\
+	    echo $(RM) $$i";" cp $(DESTDIR)$(templatedir)/$$i .;	\
 	    $(RM) $$i;						\
-	    $(LN_S) $(DESTDIR)$(templatedir)/$$i .;				\
+	    cp $(DESTDIR)$(templatedir)/$$i .;				\
 	  done)
 uninstall-local:
 	@(echo cd $(DESTDIR)$(configdir);			\
