diff -ur OpenBLAS-0.3.13.orig/driver/others/Makefile OpenBLAS-0.3.13/driver/others/Makefile
--- OpenBLAS-0.3.13.orig/driver/others/Makefile	2020-12-27 09:18:47.133825725 -0500
+++ OpenBLAS-0.3.13/driver/others/Makefile	2020-12-27 09:22:06.676660536 -0500
@@ -1,7 +1,7 @@
 TOPDIR	= ../..
 include ../../Makefile.system
 
-COMMONOBJS	 = memory.$(SUFFIX) xerbla.$(SUFFIX) c_abs.$(SUFFIX) z_abs.$(SUFFIX) openblas_set_num_threads.$(SUFFIX) openblas_get_num_threads.$(SUFFIX) openblas_get_num_procs.$(SUFFIX) openblas_get_config.$(SUFFIX) openblas_get_parallel.$(SUFFIX) openblas_error_handle.$(SUFFIX) openblas_env.$(SUFFIX)
+COMMONOBJS	 = memory.$(SUFFIX) c_abs.$(SUFFIX) z_abs.$(SUFFIX) openblas_set_num_threads.$(SUFFIX) openblas_get_num_threads.$(SUFFIX) openblas_get_num_procs.$(SUFFIX) openblas_get_config.$(SUFFIX) openblas_get_parallel.$(SUFFIX) openblas_error_handle.$(SUFFIX) openblas_env.$(SUFFIX)
 
 #COMMONOBJS	+= slamch.$(SUFFIX) slamc3.$(SUFFIX) dlamch.$(SUFFIX)  dlamc3.$(SUFFIX)
 
@@ -88,29 +88,26 @@
 
 ifeq ($(DYNAMIC_ARCH), 1)
 ifeq ($(ARCH),arm64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_arm64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_arm64.$(SUFFIX)
 else
 ifeq ($(ARCH),power)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_power.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_power.$(SUFFIX)
 else
 ifeq ($(ARCH),zarch)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_zarch.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_zarch.$(SUFFIX)
 else
 ifeq ($(ARCH),mips64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_mips64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_mips64.$(SUFFIX)
 else
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic.$(SUFFIX)
 endif
 endif
 endif
 endif
 else
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) parameter.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) parameter.$(SUFFIX)
 endif
 
-xerbla.$(SUFFIX) : xerbla.c
-	$(CC) $(CFLAGS) -c $< -o $(@F)
-
 dynamic.$(SUFFIX) : dynamic.c
 	$(CC) $(CFLAGS) -c $< -o $(@F)
 
@@ -198,9 +195,6 @@
 mulx.$(SUFFIX) : $(ARCH)/mulx.c
 	$(CC) $(CFLAGS) -c -DXDOUBLE -UCOMPLEX $< -o $(@F)
 
-xerbla.$(PSUFFIX) : xerbla.c
-	$(CC) $(PFLAGS) -c $< -o $(@F)
-
 parameter.$(PSUFFIX) : parameter.c ../../param.h
 	$(CC) $(PFLAGS) -c $< -o $(@F)
 
