diff -ur texinfo-6.0.orig/info/pcterm.c texinfo-6.0/info/pcterm.c
--- texinfo-6.0.orig/info/pcterm.c	2015-12-10 09:00:52.963609902 -0500
+++ texinfo-6.0/info/pcterm.c	2015-12-10 09:03:18.982503065 -0500
@@ -1486,7 +1486,7 @@
 }
 
 /* These should never be called, but they make the linker happy.  */
-
+#ifndef __MINGW32__
 int       tputs (const char *a, int b, int (*c)(int))
 {
   perror ("tputs"); return 0; /* here and below, added dummy retvals */
@@ -1526,3 +1526,4 @@
 {
   perror ("tcsetattr"); return 0;
 }
+#endif
diff -ur texinfo-6.0.orig/info/echo-area.c texinfo-6.0/info/echo-area.c
--- texinfo-6.0.orig/info/echo-area.c	2015-12-10 09:20:00.142339683 -0500
+++ texinfo-6.0/info/echo-area.c	2015-12-11 08:22:54.434892634 -0500
@@ -1529,7 +1529,7 @@
 static void
 pause_or_input (void)
 {
-#ifdef FD_SET
+#if defined(FD_SET) && !defined(__MINGW32__)
   struct timeval timer;
   fd_set readfds;
 
diff -ur texinfo-6.0.orig/info/man.c texinfo-6.0/info/man.c
--- texinfo-6.0.orig/info/man.c	2015-12-10 09:20:00.149339822 -0500
+++ texinfo-6.0/info/man.c	2015-12-11 08:22:07.250981195 -0500
@@ -437,7 +437,7 @@
   int bsize = 0;
   int bindex = 0;
   int select_result;
-#if defined (FD_SET)
+#if defined (FD_SET) && !defined(__MINGW32__)
   fd_set read_fds;
 
   timeout.tv_sec = 15;
diff -ur texinfo-6.0.orig/info/session.c texinfo-6.0/info/session.c
--- texinfo-6.0.orig/info/session.c	2015-12-10 09:20:00.148339802 -0500
+++ texinfo-6.0/info/session.c	2015-12-11 08:21:09.519866019 -0500
@@ -450,7 +450,7 @@
     space_avail = sizeof (info_input_buffer) - (push_index - pop_index);
 
   /* If we can just find out how many characters there are to read, do so. */
-#if defined (FIONREAD)
+#if defined (FIONREAD) && !defined(__MINGW32__)
   {
     ioctl (tty, FIONREAD, &chars_avail);
 
@@ -644,7 +644,7 @@
       if (in_map && pop_index == push_index)
         {
           int ready = 0;
-#if defined (FD_SET)
+#if defined (FD_SET) && !defined(__MINGW32__)
           struct timeval timer, *timerp = 0;
           fd_set readfds;
 
diff -ur texinfo-6.0.orig/info/session.c texinfo-6.0/info/session.c
--- texinfo-6.0.orig/info/session.c	2015-12-11 08:35:45.030871874 -0500
+++ texinfo-6.0/info/session.c	2015-12-11 08:37:57.463463567 -0500
@@ -5046,7 +5046,7 @@
       ready = 1;
       if (!info_any_buffered_input_p ())
         {
-#if defined (FD_SET)
+#if defined (FD_SET) && !defined(__MINGW32__)
           struct timeval timer;
           fd_set readfds;
 
diff -ur texinfo-6.0.orig/tp/texi2any.pl texinfo-6.0/tp/texi2any.pl
--- texinfo-6.0.orig/tp/texi2any.pl	2015-12-12 14:09:04.951513297 -0500
+++ texinfo-6.0/tp/texi2any.pl	2015-12-12 14:12:59.763522012 -0500
@@ -50,7 +50,7 @@
   my ($real_command_name, $command_directory, $command_suffix) 
      = fileparse($0, '.pl');
 
-  my $datadir = '@datadir@';
+  my $datadir = '/usr/share'
   my $package = '@PACKAGE@';
   my $updir = File::Spec->updir();
 
