diff -ur nsis-3.05-src.orig/Docs/src/bin/halibut/halibut.h nsis-3.05-src/Docs/src/bin/halibut/halibut.h
--- nsis-3.05-src.orig/Docs/src/bin/halibut/halibut.h	2020-07-18 14:38:35.641115880 -0400
+++ nsis-3.05-src/Docs/src/bin/halibut/halibut.h	2020-07-18 14:40:38.515275197 -0400
@@ -269,7 +269,7 @@
  * version.c
  */
 void initversionstring(void);
-const char *const version;
+extern const char *const version;
 
 /*
  * misc.c
