diff -rNu a/src/Makefile b/src/Makefile
--- a/src/Makefile	2012-05-08 08:06:50.000000000 -0400
+++ b/src/Makefile	2013-01-11 22:50:36.000000000 -0500
@@ -1,6 +1,6 @@
 sinclude Makeconf
 
-HDF5_LIBS := $(shell  grep "\#define OCTAVE_CONF_HDF5_LIBS"  $(shell  $(MKOCTFILE) -p OCTINCLUDEDIR)/oct-conf.h | sed 's/^.*LIBS //;s/"//g'  )
+HDF5_LIBS := $(shell  grep "\#define OCTAVE_CONF_HDF5_LIBS"  '$(shell  $(MKOCTFILE) -p OCTINCLUDEDIR)/oct-conf.h' | sed 's/^.*LIBS //;s/"//g'  )
 
 GALOISTARGET = gf.oct
 GALOISSOURCES = galois.cc galois-def.cc galoisfield.cc gf.cc \
