diff -r d85ae875cc23 src/bytea2var.cc
--- a/src/bytea2var.cc	Tue Mar 28 11:33:49 2017 +0200
+++ b/src/bytea2var.cc	Thu Jul 13 16:47:45 2017 -0400
@@ -94,7 +94,11 @@
 
       flt_fmt = mopt_digit_to_float_format (tmp);
 
+#ifdef HAVE_OCTAVE_MACH_INFO
+      if (flt_fmt == octave::mach_info::flt_fmt_unknown)
+#else
       if (flt_fmt == oct_mach_info::flt_fmt_unknown)
+#endif
         {
           error ("%s: unrecognized binary format", fname.c_str ());
 
diff -r d85ae875cc23 src/config.h.in
--- a/src/config.h.in	Tue Mar 28 11:33:49 2017 +0200
+++ b/src/config.h.in	Thu Jul 13 16:47:45 2017 -0400
@@ -4,3 +4,6 @@
 /* Define as 1 if liboctinterp has
 'verror (octave_execution_exception&, const char *, va_list)'. */
 #undef HAVE_OCTAVE_VERROR_ARG_EXC
+
+/* Define as 1 if we have octave::mach_info */
+#undef HAVE_OCTAVE_MACH_INFO
diff -r d85ae875cc23 src/configure.ac
--- a/src/configure.ac	Tue Mar 28 11:33:49 2017 +0200
+++ b/src/configure.ac	Thu Jul 13 16:47:45 2017 -0400
@@ -102,6 +102,15 @@
                     [verror (e, "test", args);]])],
   [AC_DEFINE([HAVE_OCTAVE_VERROR_ARG_EXC], 1)])
 
+## look for newer octave::mach_info in octave
+LIBS="-loctave $LIBS"
+AC_LINK_IFELSE(
+  [AC_LANG_PROGRAM([[#include <octave/oct.h>]
+                    [#include <octave/byte-swap.h>]],
+                   [[octave::mach_info::native_float_format();]])],
+  [AC_DEFINE([HAVE_OCTAVE_MACH_INFO], 1)])
+
+
 LIBS=$TLIBS
 LDFLAGS=$TLDFLAGS
 CXXFLAGS=$TCXXFLAGS
