diff -uNr a/src/isdicom.cpp b/src/isdicom.cpp
--- a/src/isdicom.cpp	2018-04-08 19:19:05.638729302 -0400
+++ b/src/isdicom.cpp	2018-04-25 10:44:45.240179042 -0400
@@ -32,7 +32,7 @@
   reader.SetFileName (filename.c_str ());
   // gdcm::Reader.Read() will return false if the file does not exists but
   // also prints to stderr so we check it first.
-  return file_stat (filename).exists () && reader.Read ();
+  return octave::sys::file_stat (filename).exists () && reader.Read ();
 }
 
 DEFUN_DLD (isdicom, args, ,
