diff -ur fits.orig/src/Makefile fits/src/Makefile
--- fits.orig/src/Makefile	2014-01-12 00:19:31.000000000 -0500
+++ fits/src/Makefile	2014-01-12 00:52:57.000000000 -0500
@@ -1,4 +1,4 @@
-MKOCT_FLAGS = $(shell pkg-config --libs cfitsio)
+MKOCT_FLAGS = $(shell pkg-config --libs --cflags cfitsio)
 
 all: read_fits_image.oct save_fits_image.oct save_fits_image_multi_ext.oct
 
diff -ur fits.orig/src/read_fits_image.cc fits/src/read_fits_image.cc
--- fits.orig/src/read_fits_image.cc	2014-01-12 00:19:31.000000000 -0500
+++ fits/src/read_fits_image.cc	2014-01-12 00:51:26.000000000 -0500
@@ -136,7 +136,7 @@
   }
   //std::cerr << "read_sz: " << read_sz << std::endl;
 
-  #if OCTAVE_API_VERSION_NUMBER >= 45 
+  #if ! defined OCTAVE_API_VERSION_NUMBER || OCTAVE_API_VERSION_NUMBER >= 45 
     MArray<double> image_data( dims ); // a octace double-type array
   #else
     MArrayN<double> image_data( dims ); // a octave double-type array
