diff -ur fl-core/src/Makefile fl-core.new/src/Makefile
--- fl-core/src/Makefile	2011-02-06 14:09:03.000000000 -0500
+++ fl-core.new/src/Makefile	2015-03-20 13:45:48.577323118 -0400
@@ -1,15 +1,23 @@
 OCT = fl_compose.oct
 SRC := $(OCT:.oct=.cc)
 BASE := $(OCT:.oct=)
-ifdef COMSPEC 
+
+MKOCTFILE ?= mkoctfile
+OCTAVE_CONFIG ?= octave-config
+
+ifneq (,$(findstring mingw,$(shell $(OCTAVE_CONFIG) -p CANONICAL_HOST_TYPE)))
+	ADDPARAM := -lpthread
+else
+ ifdef COMSPEC 
 	ADDPARAM := -lpthreadVC2
+ endif
 endif
 
 .phony: all
 all: $(OCT)
 
 %.oct: %.cc
-	mkoctfile -s $< $(ADDPARAM)
+	$(MKOCTFILE) -s $< $(ADDPARAM)
 
 .phony: clean
 clean: 
