diff -uNr a/DESCRIPTION b/DESCRIPTION
--- a/DESCRIPTION	2017-03-29 09:52:36.556868803 -0400
+++ b/DESCRIPTION	2018-04-24 13:13:41.074914449 -0400
@@ -1,14 +1,10 @@
 Name: geometry
 Version: 3.0.0
 Date: 27-03-2017
-Author: Juan Pablo Carbajal <ajuanpi+dev@gmail.com>,
-Philip Neuhius <prnienhuis@users.sf.net>,
-Simeon Simeonov <sss41@cam.ac.uk>,
-David Legland <david.legland@nantes.inra.fr>,
+Author: Juan Pablo Carbajal <ajuanpi+dev@gmail.com>, Philip Neuhius <prnienhuis@users.sf.net>, Simeon Simeonov <sss41@cam.ac.uk>, David Legland <david.legland@nantes.inra.fr>
 Maintainer: Juan Pablo Carbajal <ajuanpi+dev@gmail.com>
 Title: Computational Geometry
-Description: Library for geometric computing extending MatGeom functions.
-Useful to create, transform, manipulate and display geometric primitives.
+Description: Library for geometric computing extending MatGeom functions.  Useful to create, transform, manipulate and display geometric primitives.
 Depends: octave (>= 4.0.1)
 Autoload: no
 License: GPLv3+, FreeBSD, Boost v1.0
