diff -ur interval-3.0.0.orig/src/Makefile interval-3.0.0/src/Makefile
--- interval-3.0.0.orig/src/Makefile	2017-08-22 12:31:47.602400636 -0400
+++ interval-3.0.0/src/Makefile	2017-08-22 12:33:50.151855098 -0400
@@ -29,6 +29,8 @@
 CONF_FLAG_SSE2 ?=--enable-sse2
 endif
 
+CRLIBM_CONFIG_FLAGS ?=
+
 all: $(OBJ)
 
 ## crlibm api oct-file
@@ -48,7 +50,7 @@
 crlibm/crlibm_private.o: crlibm/crlibm_config.h
 	$(MAKE) -C crlibm CFLAGS+="$(shell $(MKOCTFILE) -p CPICFLAG) -Wno-div-by-zero -Wno-unused-variable -Wno-unused-but-set-variable"
 crlibm/crlibm_config.h:
-	(cd crlibm && ./configure --disable-dependency-tracking $(CONF_FLAG_SSE2))
+	(cd crlibm && ./configure --disable-dependency-tracking $(CONF_FLAG_SSE2) $(CRLIBM_CONFIG_FLAGS))
 
 ## GNU MPFR api oct-files
 mpfr_matrix_mul_d.oct mpfr_matrix_sqr_d.oct : mpfr_%.oct: mpfr_%.cc mpfr_commons.h
