diff -ur interval-3.1.0.orig/src/Makefile interval-3.1.0/src/Makefile
--- interval-3.1.0.orig/src/Makefile	2017-12-21 12:38:19.524166411 -0500
+++ interval-3.1.0/src/Makefile	2017-12-21 12:43:27.693376135 -0500
@@ -31,6 +31,8 @@
 CONF_FLAG_SSE2 ?=--enable-sse2
 endif
 
+CRLIBM_CONFIG_FLAGS ?=
+
 all: $(OBJ)
 
 ## crlibm api oct-file
@@ -50,7 +52,7 @@
 $(BUNDLED_CRLIBM_DIR)/crlibm_private.o: $(BUNDLED_CRLIBM_DIR)/crlibm_config.h
 	$(MAKE) -C $(BUNDLED_CRLIBM_DIR) CFLAGS+="$(shell $(MKOCTFILE) -p CPICFLAG) -Wno-div-by-zero -Wno-unused-variable -Wno-unused-but-set-variable"
 $(BUNDLED_CRLIBM_DIR)/crlibm_config.h:
-	(cd $(BUNDLED_CRLIBM_DIR) && ./configure --disable-dependency-tracking $(CONF_FLAG_SSE2))
+	(cd $(BUNDLED_CRLIBM_DIR) && ./configure --disable-dependency-tracking $(CONF_FLAG_SSE2) $(CRLIBM_CONFIG_FLAGS))
 
 ## GNU MPFR api oct-files
 mpfr_matrix_mul_d.oct mpfr_matrix_sqr_d.oct : mpfr_%.oct: mpfr_%.cc mpfr_commons.h
