diff -uNr a/src/gsvd.cc b/src/gsvd.cc
--- a/src/gsvd.cc	2015-01-24 14:54:11.000000000 -0500
+++ b/src/gsvd.cc	2018-04-09 14:25:44.604821251 -0400
@@ -23,8 +23,8 @@
 
 #include "defun-dld.h"
 #include "error.h"
-#include "gripes.h"
-#include "oct-obj.h"
+#include "errwarn.h"
+#include "ovl.h"
 #include "pr-output.h"
 #include "utils.h"
 
@@ -211,7 +211,7 @@
                         ? GSVD::sigma_only
                         : (nargout > 5) ? GSVD::std : GSVD::economy );
 
-      if (argA.is_real_type () && argB.is_real_type ())
+      if (argA.isreal () && argB.isreal ())
         {
           Matrix tmpA = argA.matrix_value ();
           Matrix tmpB = argB.matrix_value ();
@@ -253,7 +253,7 @@
                 }
             }
         }
-      else if (argA.is_complex_type () || argB.is_complex_type ())
+      else if (argA.iscomplex () || argB.iscomplex ())
         {
           ComplexMatrix ctmpA = argA.complex_matrix_value ();
           ComplexMatrix ctmpB = argB.complex_matrix_value ();
@@ -296,8 +296,8 @@
         }
       else
         {
-          gripe_wrong_type_arg ("gsvd", argA);
-          gripe_wrong_type_arg ("gsvd", argB);
+          err_wrong_type_arg ("gsvd", argA);
+          err_wrong_type_arg ("gsvd", argB);
           return retval;
         }
     }
