diff -ur miscellaneous-1.2.1.orig/inst/units.m miscellaneous-1.2.1/inst/units.m
--- miscellaneous-1.2.1.orig/inst/units.m	2018-07-11 12:44:26.038777666 -0400
+++ miscellaneous-1.2.1/inst/units.m	2018-07-11 12:45:50.033750024 -0400
@@ -107,7 +107,7 @@
 
 function fpath = check_units ()
   ## See bug #38270 about why we're checking this way.
-  fpath = file_in_path (getenv ("PATH"), sprintf ("units%s", octave_config_info ("EXEEXT")));
+  fpath = file_in_path (getenv ("PATH"), sprintf ("units%s", __octave_config_info__ ("EXEEXT")));
   if (isempty (fpath))
     error ("units: %s\nVerify that GNU units is installed in the current path.",
            rawoutput);
diff -ur miscellaneous-1.2.1.orig/src/partint.cc miscellaneous-1.2.1/src/partint.cc
--- miscellaneous-1.2.1.orig/src/partint.cc	2019-01-02 12:13:28.286461534 -0500
+++ miscellaneous-1.2.1/src/partint.cc	2019-01-02 12:14:43.200262618 -0500
@@ -77,7 +77,7 @@
         error("partcnt accepts exactly one argument");
         return r; 
     }
-    if ( ! args(0).is_numeric_type()) {
+    if ( ! args(0).isnumeric()) {
         error("partcnt only accepts a numeric argument");
         return r;
     }
@@ -181,7 +181,7 @@
     int nargin = args.length ();
     if (nargin != 1 || 
         ! args(0).is_scalar_type() ||
-        ! args(0).is_numeric_type()
+        ! args(0).isnumeric()
         ) {
         error("partint only accepts one scalar positive integer argument");
         return r;
diff -ur miscellaneous-1.2.1.orig/src/sample.cc miscellaneous-1.2.1/src/sample.cc
--- miscellaneous-1.2.1.orig/src/sample.cc	2019-01-02 12:18:26.373626834 -0500
+++ miscellaneous-1.2.1/src/sample.cc	2019-01-02 12:19:15.234801067 -0500
@@ -123,7 +123,7 @@
 	/* as in the code from "Programming Pearls" */
 	IntSetBins2 S(m, n);
 	while (S.size() < m)
-		S.insert(floor(oct_randu()*n)); // use Octave's uniform RNG
+		S.insert(floor(octave::rand_uniform<double> ()*n)); // use Octave's uniform RNG
 	S.report(s.fortran_vec());
 	return octave_value (s);
 }
