diff -ur mapping-1.2.1.orig/src/configure mapping-1.2.1/src/configure
--- mapping-1.2.1.orig/src/configure	2016-02-08 12:49:21.906234446 -0500
+++ mapping-1.2.1/src/configure	2016-02-08 12:50:26.136276230 -0500
@@ -1810,8 +1810,8 @@
   OCTAVE=octave
 fi
 
-HAS_RAD2DEG=`$OCTAVE -qf --eval "printf ('%i', exist ('rad2deg'))"`
-HAS_DEG2RAD=`$OCTAVE -qf --eval "printf ('%i', exist ('deg2rad'))"`
+HAS_RAD2DEG=0 #`$OCTAVE -qf --eval "printf ('%i', exist ('rad2deg'))"`
+HAS_DEG2RAD=0 #`$OCTAVE -qf --eval "printf ('%i', exist ('deg2rad'))"`
 
 if test "$HAS_RAD2DEG" == 0; then
   ac_config_files="$ac_config_files rad2deg.m"
diff -ur mapping-1.2.1.orig/src/configure.ac mapping-1.2.1/src/configure.ac
--- mapping-1.2.1.orig/src/configure.ac	2016-02-08 12:49:21.906234446 -0500
+++ mapping-1.2.1/src/configure.ac	2016-02-08 12:50:01.067650216 -0500
@@ -11,8 +11,8 @@
   OCTAVE=octave
 fi
 
-HAS_RAD2DEG=`$OCTAVE -qf --eval "printf ('%i', exist ('rad2deg'))"`
-HAS_DEG2RAD=`$OCTAVE -qf --eval "printf ('%i', exist ('deg2rad'))"`
+HAS_RAD2DEG=0 #`$OCTAVE -qf --eval "printf ('%i', exist ('rad2deg'))"`
+HAS_DEG2RAD=0 #`$OCTAVE -qf --eval "printf ('%i', exist ('deg2rad'))"`
 
 if test "$HAS_RAD2DEG" == 0; then
   AC_CONFIG_FILES([rad2deg.m])
