diff -ur nan-3.0.2.orig/src/Makefile nan-3.0.2/src/Makefile
--- nan-3.0.2.orig/src/Makefile	2016-08-03 10:30:35.433478948 -0400
+++ nan-3.0.2/src/Makefile	2016-08-03 10:42:48.629174523 -0400
@@ -31,8 +31,9 @@
 # Instead of building "mex shortpath.c" and "mex uigetpath.c", I used empty m-functions within argout=argin; 
 ####################################################
 MKOCTFILE ?= mkoctfile$(OCTAVE_VERSION)
-CC      ?= gcc
-CXX     ?= g++
+OCTAVE_CONFIG ?= octave-config
+CC      ?= $(shell $(MKOCTFILE) -p CC)
+CXX     ?= $(shell $(MKOCTFILE) -p CXX)
 CFLAGS += -Wall -Wextra -Wconversion -O2
 OCTMEX	= $(MKOCTFILE) --mex
 RM      = rm
@@ -41,9 +42,10 @@
 # https://wiki.debian.org/HardeningWalkthrough#Handling_dpkg-buildflags_in_your_upstream_build_system
 CFLAGS += $(CPPFLAGS)
 
-ifeq ($(OS),Windows_NT)
+HOSTTYPE := $(shell $(OCTAVE_CONFIG) -p CANONICAL_HOST_TYPE)
+ifneq (,$(findstring mingw,$(HOSTTYPE)))
   # CC need to be defined on Windows, see also http://savannah.gnu.org/bugs/?47559
-  CC = gcc
+  CC = $(shell $(MKOCTFILE) -p CC)
 else ifeq (Darwin,$(shell uname))
   # no OpenMP on MacOSX, neither for Matlab nor Octave
 else
@@ -93,14 +95,14 @@
 #	Octave, MATLAB on Linux 
 #########################################################
 svm%_mex.mex: svm%_mex.cpp svm.o svm_model_octave.o
-	env CC=$(CXX) $(OCTMEX) "$<" svm.o svm_model_octave.o
+	env CC="$(CXX)" $(OCTMEX) "$<" svm.o svm_model_octave.o
 svm%_mex.$(MEX_EXT): svm%_mex.cpp svm.o svm_model_matlab.o
 	$(MATMEX) "$<" svm.o svm_model_matlab.o
 %.$(MEX_EXT): %.cpp
 	$(MATMEX) "$<"
 
 svm_model_octave.o: svm_model_matlab.c 
-	env CC=$(CC) $(MKOCTFILE) -o "$@" -c "$<"
+	env CC="$(CC)" $(MKOCTFILE) -o "$@" -c "$<"
 svm_model_matlab.o: svm_model_matlab.c 
 	$(CXX) $(CFLAGS) -I $(MATLABDIR)/extern/include -o "$@" -c "$<"  
 svm.o: svm.cpp
@@ -111,7 +113,7 @@
 	$(MATMEX)  -lblas train.c tron.o linear.o linear_model_matlab.o 
 	#$(MATMEX) -lblas predict.c tron.o linear.o linear_model_matlab.o
 train.mex predict.mex: train.c tron.o linear.o linear_model_matlab.c
-	env CC=$(CXX) $(OCTMEX) -lblas train.c tron.o linear.o linear_model_matlab.c
+	env CC="$(CXX)" $(OCTMEX) -lblas train.c tron.o linear.o linear_model_matlab.c
 linear.o: linear.cpp
 	$(CXX) $(CFLAGS) -c linear.cpp
 tron.o: tron.cpp tron.h
