diff -ur optiminterp-0.3.5.orig/src/optiminterp.cc optiminterp-0.3.5/src/optiminterp.cc
--- optiminterp-0.3.5.orig/src/optiminterp.cc	2019-01-02 14:57:31.522221800 -0500
+++ optiminterp-0.3.5/src/optiminterp.cc	2019-01-02 14:58:27.823641722 -0500
@@ -130,11 +130,6 @@
 	    ( n,nf,gn,on,nparam,ox.fortran_vec(),of.fortran_vec(),ovar.fortran_vec(),
               param.fortran_vec(),m,gx.fortran_vec(),gf.fortran_vec(),gvar.fortran_vec()));
 
-  if (f77_exception_encountered) {
-      error ("unrecoverable error in optiminterp");
-      return retval;
-  }
-
   gf = gf.transpose();
 
   retval(0) = gf;
