diff -ur octave-sockets-1.2.0.orig/src/Makefile octave-sockets-1.2.0/src/Makefile
--- octave-sockets-1.2.0.orig/src/Makefile	2015-03-20 10:24:40.824141293 -0400
+++ octave-sockets-1.2.0/src/Makefile	2015-03-20 11:31:54.325799984 -0400
@@ -8,12 +8,15 @@
 SRC := $(OCT:.oct=.cc)
 
 MKOCTFILE ?= mkoctfile -Wall
+OCTAVECONFIG ?= octave-config
 OCTAVE ?= octave
 
+CANONICAL_HOST_TYPE := $(shell $(OCTAVECONFIG) -p CANONICAL_HOST_TYPE))
+
 #The following is necessary to get the sockets package working in Windows.
 #It has been tried on Win7 and XP, in Octave 3.8.0 using mxe-octave (mingw)
 EXTRALIBS := 
-ifeq ($(OS),Windows_NT)
+ifneq (,$(findstring mingw,$(CANONICAL_HOST_TYPE)))
   EXTRALIBS := -lws2_32
 endif
 
