diff -ur signal-1.4.0.old/src/medfilt1.cc signal-1.4.0/src/medfilt1.cc
--- signal-1.4.0.old/src/medfilt1.cc	2019-01-02 12:00:17.695424340 -0500
+++ signal-1.4.0/src/medfilt1.cc	2019-01-02 12:01:01.900491183 -0500
@@ -324,7 +324,7 @@
 
   if (nargin >= 2)
     {
-      if (args(1).is_numeric_type ())
+      if (args(1).isnumeric ())
         {
           if (args(1).numel () != 1 || args(1).iscomplex ())
             error ("medfilt1: N must be a real scalar");
@@ -336,7 +336,7 @@
                args(1).type_name ().c_str ());
     if (nargin >= 4)
       {
-        if (args(3).is_numeric_type ())
+        if (args(3).isnumeric ())
           {
             if (args(3).numel () != 1)
               error ("medfilt1: DIM must be a scalar");
diff -ur signal-1.4.0.orig/src/upfirdn.cc signal-1.4.0/src/upfirdn.cc
--- signal-1.4.0.orig/src/upfirdn.cc	2019-01-02 12:09:32.072780390 -0500
+++ signal-1.4.0/src/upfirdn.cc	2019-01-02 12:09:52.581273776 -0500
@@ -130,7 +130,7 @@
     }
 
   // Do the dispatching
-  if (args (0).is_real_type ())
+  if (args (0).isreal ())
     {
       Matrix x = args (0).matrix_value ();
       if (error_state)
