diff -ur strings-1.2.0.orig/src/Makefile strings-1.2.0/src/Makefile
--- strings-1.2.0.orig/src/Makefile	2015-06-08 13:43:45.551556353 -0400
+++ strings-1.2.0/src/Makefile	2015-06-08 13:45:22.693478306 -0400
@@ -1,11 +1,7 @@
 OCTAVE ?= octave
 MKOCTFILE ?= mkoctfile -Wall
 
-PCRE_SWITCHES := $(shell $(OCTAVE) \
-	--no-gui --no-init-file --no-site-file --silent --no-history \
-	--eval 'disp (octave_config_info ("PCRE_LIBS"));' \
-	--eval 'disp (octave_config_info ("PCRE_CPPFLAGS"));' \
-	)
+PCRE_SWITCHES := $(shell pcre-config --libs)
 
 pcregexp.oct: %.oct: %.cc
 	$(MKOCTFILE) $(PCRE_SWITCHES) -o $@ $<
