diff -ur windows-1.3.0.orig/src/__COM__.cc windows-1.3.0/src/__COM__.cc
--- windows-1.3.0.orig/src/__COM__.cc	2019-01-02 10:47:50.386998529 -0500
+++ windows-1.3.0/src/__COM__.cc	2019-01-02 10:48:25.859871218 -0500
@@ -69,7 +69,7 @@
   return std::string (&tmp[0], len);
 }
 
-class octave_com_object : public octave_base_value, public octave_auto_shlib
+class octave_com_object : public octave_base_value
 {
 public:
   // Constructors
