diff -ur pstoedit-3.74-1/configure.ac pstoedit-3.74/configure.ac
--- pstoedit-3.74-1/configure.ac	2019-08-12 17:00:05.383739915 -0400
+++ pstoedit-3.74/configure.ac	2019-08-12 17:01:34.890067312 -0400
@@ -44,8 +44,8 @@
 dnl fi
 
 dnl Check Ghostscript
-AC_CHECK_PROGS([GS],[gs],[false])
-AS_IF([test x"${GS}" = xfalse],[AC_MSG_ERROR([** Cannot find ghostscript.  If it is already installed, Check PATH.])])
+dnl AC_CHECK_PROGS([GS],[gs],[false])
+dnl AS_IF([test x"${GS}" = xfalse],[AC_MSG_ERROR([** Cannot find ghostscript.  If it is already installed, Check PATH.])])
 
 PKG_PROG_PKG_CONFIG
 
diff -ur pstoedit-3.74-1/src/Makefile.am pstoedit-3.74/src/Makefile.am
--- pstoedit-3.74-1/src/Makefile.am	2019-08-12 17:00:05.395740228 -0400
+++ pstoedit-3.74/src/Makefile.am	2019-08-12 17:02:29.107472650 -0400
@@ -32,7 +32,7 @@
 	${LIBEMF_CFLAGS}			\
 	${LIBPNG_CFLAGS}			\
 	${LIBZIP_CFLAGS}			\
-        -DDEFAULTGS=$(GS)			\
+        -DDEFAULTGS="\"$(GS)\""			\
 	-DPSTOEDITDATADIR="\"$(pkgdatadir)\""	\
 	-DPSTOEDITLIBDIR="\"$(pkglibdir)\""	\
         ${SPECIAL_CXXFLAGS}
