diff -ur pstoedit-3.70.orig/configure.ac pstoedit-3.70/configure.ac
--- pstoedit-3.70.orig/configure.ac	2015-02-27 13:34:44.538158342 -0500
+++ pstoedit-3.70/configure.ac	2015-02-27 15:12:27.821126740 -0500
@@ -53,12 +53,12 @@
 dnl Check Ghostscript
 GS=""
 AC_PATH_PROG(GS,gs)
-if test -z "${GS}"; then
-   AC_MSG_ERROR([** Cannot find ghostscript. If it is already installed, Check PATH.])
-else
-dnl Strip off // to / that causes troubles.
-   GS=`echo $GS | sed -e 's/\/\//\//g'`	
-fi
+dnl if test -z "${GS}"; then
+dnl    AC_MSG_ERROR([** Cannot find ghostscript. If it is already installed, Check PATH.])
+dnl else
+dnl dnl Strip off // to / that causes troubles.
+dnl    GS=`echo $GS | sed -e 's/\/\//\//g'`	
+dnl fi
 
 PKG_PROG_PKG_CONFIG
 
diff -ur pstoedit-3.70.orig/src/Makefile.am pstoedit-3.70/src/Makefile.am
--- pstoedit-3.70.orig/src/Makefile.am	2015-02-27 13:34:45.004167901 -0500
+++ pstoedit-3.70/src/Makefile.am	2015-02-27 15:13:10.916976944 -0500
@@ -32,7 +32,7 @@
 	${LIBEMF_CFLAGS}			\
 	${LIBPNG_CFLAGS}			\
 	${LIBZIP_CFLAGS}			\
-        -DDEFAULTGS=$(GS)			\
+        -DDEFAULTGS="\"$(GS)\""			\
 	-DPSTOEDITDATADIR="\"$(pkgdatadir)\""	\
 	-DPSTOEDITLIBDIR="\"$(pkglibdir)\""	\
         ${SPECIAL_CXXFLAGS}
