diff -ur pstoedit-3.62.orig/src/cppcomp.h pstoedit-3.62/src/cppcomp.h
--- pstoedit-3.62.orig/src/cppcomp.h	2014-07-25 09:09:48 -0400
+++ pstoedit-3.62/src/cppcomp.h	2014-07-25 09:12:17 -0400
@@ -33,7 +33,12 @@
 # define DLLEXPORT
 #endif
 
-
+#ifdef __MINGW32__
+  #include <_mingw.h>
+  #ifdef MINGW_HAS_SECURE_API
+    #undef MINGW_HAS_SECURE_API
+  #endif
+#endif
 
 
 #ifdef _AIX
diff -ur pstoedit-3.62.orig/src/lexyy.cpp pstoedit-3.62/src/lexyy.cpp
--- pstoedit-3.62.orig/src/lexyy.cpp	2014-07-25 09:19:14 -0400
+++ pstoedit-3.62/src/lexyy.cpp	2014-07-25 09:20:00 -0400
@@ -14,6 +14,7 @@
 #endif
 
 /* First, we deal with  platform-specific or compiler-specific issues. */
+#include "cppcomp.h"
 
 /* begin standard C headers. */
 #include <stdio.h>
