diff -ur pstoedit-3.74.orig/othersrc/gsdllinc/wgsver.c pstoedit-3.74/othersrc/gsdllinc/wgsver.c
--- pstoedit-3.74.orig/othersrc/gsdllinc/wgsver.c	2019-08-12 16:51:30.466523053 -0400
+++ pstoedit-3.74/othersrc/gsdllinc/wgsver.c	2019-08-12 16:57:43.384024846 -0400
@@ -40,6 +40,10 @@
 #include "wgsver.h"
 #include "cppcomp.h"
 
+#ifndef KEY_WOW64_64KEY
+#define KEY_WOW64_64KEY 0x0100
+#endif
+
 /* Ghostscript may be known in the Windows Registry by
  * the following names.
  */
Only in pstoedit-3.74/othersrc/gsdllinc: wgsver.c.orig
diff -ur pstoedit-3.74.orig/src/cbstream.cpp pstoedit-3.74/src/cbstream.cpp
--- pstoedit-3.74.orig/src/cbstream.cpp	2019-08-12 16:51:30.466523053 -0400
+++ pstoedit-3.74/src/cbstream.cpp	2019-08-12 16:57:43.388024951 -0400
@@ -33,6 +33,8 @@
 // this code is only needed under WIN32
 // ...and under OS/2
 
+#include <cstdio>
+
 #include "cppcomp.h"
 
 #include "cbstream.h"
diff -ur pstoedit-3.74.orig/src/dynload.cpp pstoedit-3.74/src/dynload.cpp
--- pstoedit-3.74.orig/src/dynload.cpp	2019-08-12 16:51:30.470523154 -0400
+++ pstoedit-3.74/src/dynload.cpp	2019-08-12 16:57:43.388024951 -0400
@@ -240,7 +240,7 @@
 #elif defined(OS_WIN32_WCE)
 	DynLoader::fptr rfptr = ptr_to_fptr(GetProcAddress((HINSTANCE) handle, LPSTRtoLPWSTR(name).c_str()));	//lint !e611 //: Suspicious cast
 #elif defined(_WIN32)
-	DynLoader::fptr rfptr = ptr_to_fptr(GetProcAddress((HINSTANCE) handle, name));	//lint !e611 //: Suspicious cast
+	DynLoader::fptr rfptr = ptr_to_fptr((void*)GetProcAddress((HINSTANCE) handle, name));	//lint !e611 //: Suspicious cast
 #else
 #error "system unsupported so far"
 #endif
Only in pstoedit-3.74/src: dynload.cpp.orig
diff -ur pstoedit-3.74.orig/src/Makefile.am pstoedit-3.74/src/Makefile.am
--- pstoedit-3.74.orig/src/Makefile.am	2019-08-12 16:51:30.470523154 -0400
+++ pstoedit-3.74/src/Makefile.am	2019-08-12 16:57:43.388024951 -0400
@@ -143,7 +143,8 @@
 	psfront.h				\
 	genericints.h
 
-libpstoedit_la_LIBADD = ${CXX_STD_LIB} ${CXX_RUNTIME_LIB} -ldl -lm
+libpstoedit_la_LIBADD = ${CXX_STD_LIB} ${CXX_RUNTIME_LIB} -lm
+libpstoedit_la_CPPFLAGS = -I$(top_srcdir)/othersrc/gsdllinc
 libpstoedit_la_LDFLAGS = -no-undefined
 
 pstoedit_SOURCES = cmdmain.cpp
