diff -ur QScintilla_gpl-2.10.2.orig/lexers/LexMatlab.cpp QScintilla_gpl-2.10.2/lexers/LexMatlab.cpp
--- QScintilla_gpl-2.10.2.orig/lexers/LexMatlab.cpp	2018-01-02 08:15:29.183313917 -0500
+++ QScintilla_gpl-2.10.2/lexers/LexMatlab.cpp	2018-01-02 08:16:40.754287756 -0500
@@ -62,6 +62,7 @@
 	if (strcmp ("if", str) == 0 ||
 		strcmp ("for", str) == 0 ||
 		strcmp ("switch", str) == 0 ||
+		strcmp ("while", str) == 0 ||
 		strcmp ("try", str) == 0 ||
 		strcmp ("do", str) == 0 ||
 		strcmp ("parfor", str) == 0 ||
