diff -ur QScintilla-gpl-2.9.2.orig/Qt4Qt5/qscintilla.pro QScintilla_gpl-2.9.2/Qt4Qt5/qscintilla.pro
--- QScintilla-gpl-2.9.2.orig/Qt4Qt5/qscintilla.pro	2016-09-09 09:27:33.180610774 -0400
+++ QScintilla_gpl-2.9.2/Qt4Qt5/qscintilla.pro	2016-09-09 09:28:05.519426207 -0400
@@ -24,7 +24,7 @@
 
 TEMPLATE = lib
 TARGET = qscintilla2
-CONFIG += qt warn_off release thread exceptions hide_symbols
+CONFIG += qt warn_off release thread exceptions
 INCLUDEPATH += . ../include ../lexlib ../src
 
 !CONFIG(staticlib) {
