diff -ur QScintilla-gpl-2.8.3.orig/lexers/LexMatlab.cpp QScintilla-gpl-2.8.3/lexers/LexMatlab.cpp
--- QScintilla-gpl-2.8.3.orig/lexers/LexMatlab.cpp	2014-08-01 07:57:01.175910086 -0400
+++ QScintilla-gpl-2.8.3/lexers/LexMatlab.cpp	2014-08-01 08:01:05.815912051 -0400
@@ -213,7 +213,7 @@
 			} else if (isalpha(sc.ch)) {
 				sc.SetState(SCE_MATLAB_KEYWORD);
 			} else if (isoperator(static_cast<char>(sc.ch)) || sc.ch == '@' || sc.ch == '\\') {
-				if (sc.ch == ')' || sc.ch == ']') {
+				if (sc.ch == ')' || sc.ch == ']' || sc.ch == '}') {
 					transpose = true;
 				} else {
 					transpose = false;
