diff -ur QScintilla_gpl-2.10.1.orig/Qt4Qt5/qscintilla.pro QScintilla_gpl-2.10.1/Qt4Qt5/qscintilla.pro
--- QScintilla_gpl-2.10.1.orig/Qt4Qt5/qscintilla.pro	2017-07-05 08:05:16.558914764 -0400
+++ QScintilla_gpl-2.10.1/Qt4Qt5/qscintilla.pro	2017-07-05 08:05:34.640424452 -0400
@@ -23,7 +23,7 @@
 !win32:VERSION = 13.0.0
 
 TEMPLATE = lib
-CONFIG += qt warn_off thread exceptions hide_symbols
+CONFIG += qt warn_off release thread exceptions
 
 CONFIG(debug, debug|release) {
     mac: {
