diff -ur qt-everywhere-opensource-src-4.8.3.orig/src/dbus/dbus.pro qt-everywhere-opensource-src-4.8.3/src/dbus/dbus.pro
--- qt-everywhere-opensource-src-4.8.3.orig/src/dbus/dbus.pro	2013-07-13 21:04:11 -0400
+++ qt-everywhere-opensource-src-4.8.3/src/dbus/dbus.pro	2013-07-13 21:06:25 -0400
@@ -20,6 +20,7 @@
         QtXml
 }
 win32 { 
+    PKGCONFIG += dbus-1
     wince*:LIBS_PRIVATE += -lws2
     else:LIBS_PRIVATE += -lws2_32 \
         -ladvapi32 \
