diff -uNr a/input.c b/input.c
--- a/input.c	2010-05-30 18:33:01.000000000 -0400
+++ b/input.c	2013-04-17 23:50:34.759675631 -0400
@@ -466,7 +466,12 @@
 
 #if defined (__MINGW32__)
       if (isatty (fileno (stream)))
-	return (getch ());
+        {
+          int c = _getch ();
+          if (c == 0xe0)
+            rl_execute_next (_getch ());
+          return (c);
+        }
 #endif
       result = read (fileno (stream), &c, sizeof (unsigned char));
 
