diff -ur texinfo-5.2.orig/info/pcterm.c texinfo-5.2/info/pcterm.c
--- texinfo-5.2.orig/info/pcterm.c	2014-04-15 18:58:25.000000000 -0400
+++ texinfo-5.2/info/pcterm.c	2014-04-15 19:00:25.000000000 -0400
@@ -1299,7 +1299,7 @@
 }
 
 /* These should never be called, but they make the linker happy.  */
-
+#if 0
 int       tputs (const char *a, int b, int (*c)(int))
 {
   perror ("tputs"); return 0; /* here and below, added dummy retvals */
@@ -1339,3 +1339,4 @@
 {
   perror ("tcsetattr"); return 0;
 }
+#endif
