diff -ru a/util/texi2dvi b/util/texi2dvi
--- a/util/texi2dvi	2008-09-18 14:46:01.000000000 -0400
+++ b/util/texi2dvi	2014-02-14 13:21:50.038015947 -0500
@@ -1683,7 +1683,7 @@
 
   # If the COMMAND_LINE_FILENAME is not absolute (e.g., --debug.tex),
   # prepend `./' in order to avoid that the tools take it as an option.
-  echo "$command_line_filename" | $EGREP '^(/|[A-z]:/)' >&6 \
+  echo "$command_line_filename" | $EGREP '^(/|[A-Za-z]:/)' >&6 \
   || command_line_filename="./$command_line_filename"
 
   # See if the file exists.  If it doesn't we're in trouble since, even
