diff -ur fig2dev-3.2.6-rc.orig/fig2dev/dev/genbitmaps.c fig2dev-3.2.6-rc/fig2dev/dev/genbitmaps.c
--- fig2dev-3.2.6-rc.orig/fig2dev/dev/genbitmaps.c	2016-07-21 12:56:13.331222524 -0400
+++ fig2dev-3.2.6-rc/fig2dev/dev/genbitmaps.c	2016-07-21 13:01:11.202891975 -0400
@@ -160,7 +160,9 @@
     if (tfp != stdout)
 	fclose(tfp);
 
+#if ! defined (__WIN32__) || defined (__CYGWIN__)
     (void) signal(SIGPIPE, gs_broken_pipe);
+#endif
     if ((tfp = popen(gscom,"w" )) == 0) {
 	fprintf(stderr,"fig2dev: Can't open pipe to ghostscript\n");
 	fprintf(stderr,"command was: %s\n", gscom);
@@ -191,7 +193,9 @@
 	    fprintf(stderr,"command was: %s\n", gscom);
 	    return -1;
 	}
+#if ! defined (__WIN32__) || defined (__CYGWIN__)
 	(void) signal(SIGPIPE, SIG_DFL);
+#endif
 
 	/* all ok so far */
 	status = 0;
diff -ur fig2dev-3.2.6-rc.orig/fig2dev/dev/genpdf.c fig2dev-3.2.6-rc/fig2dev/dev/genpdf.c
--- fig2dev-3.2.6-rc.orig/fig2dev/dev/genpdf.c	2016-07-21 13:04:33.941910210 -0400
+++ fig2dev-3.2.6-rc/fig2dev/dev/genpdf.c	2016-07-21 13:05:54.091731415 -0400
@@ -56,7 +56,9 @@
     sprintf(gscom,
 	 "gs -q -dNOPAUSE -sAutoRotatePages=None -dAutoFilterColorImages=false -dColorImageFilter=/DCTEncode -sDEVICE=pdfwrite -dPDFSETTINGS=/prepress -sOutputFile=\'%s\' - -c quit",
 		ofile);
+#if !defined (__WIN32__) || defined (__CYGWIN__)
     (void) signal(SIGPIPE, gs_broken_pipe);
+#endif
     if ((tfp = popen(gscom,"w" )) == 0) {
 	fprintf(stderr,"fig2dev: Can't open pipe to ghostscript\n");
 	fprintf(stderr,"command was: %s\n", gscom);
@@ -82,7 +84,9 @@
 	    fprintf(stderr,"command was: %s\n", gscom);
 	    return -1;
 	}
+#if !defined (__WIN32__) || defined (__CYGWIN__)
 	(void) signal(SIGPIPE, SIG_DFL);
+#endif
 
 	/* all ok so far */
 
diff -ur fig2dev-3.2.6-rc.orig/fig2dev/dev/genps.c fig2dev-3.2.6-rc/fig2dev/dev/genps.c
--- fig2dev-3.2.6-rc.orig/fig2dev/dev/genps.c	2016-07-21 13:13:08.193346910 -0400
+++ fig2dev-3.2.6-rc/fig2dev/dev/genps.c	2016-07-21 13:14:08.520459655 -0400
@@ -120,8 +120,10 @@
 static bool ellipse_exist(F_compound *ob);
 static bool approx_spline_exist(F_compound *ob);
 static void draw_gridline(float x1, float y1, float x2, float y2);
+#ifdef HAVE_X11_XPM_H
 static void convert_xpm_colors(unsigned char cmap[3][MAXCOLORMAPSIZE],
 		XpmColor *coltabl, int ncols);
+#endif
 static void genps_itp_spline(F_spline *s);
 static void genps_ctl_spline(F_spline *s);
 
diff -ur fig2dev-3.2.6-rc.orig/fig2dev/lib/realloc.c fig2dev-3.2.6-rc/fig2dev/lib/realloc.c
--- fig2dev-3.2.6-rc.orig/fig2dev/lib/realloc.c	2016-07-21 13:17:46.898252242 -0400
+++ fig2dev-3.2.6-rc/fig2dev/lib/realloc.c	2016-07-21 13:23:33.706176197 -0400
@@ -37,6 +37,8 @@
 void *
 rpl_realloc(void *p, size_t n)
 {
+	void * result;
+
 	if (n == 0) {
 		n = 1;
 		free(p);
@@ -49,6 +51,7 @@
 		result = malloc(n);
 	} else {
 		result = realloc(p, n);
+        }
 
 	return result;
 }
Only in fig2dev-3.2.6-rc/fig2dev/dev: .genemf.c.swp
diff -ur fig2dev-3.2.6-rc.orig/fig2dev/dev/readpng.c fig2dev-3.2.6-rc/fig2dev/dev/readpng.c
--- fig2dev-3.2.6-rc.orig/fig2dev/dev/readpng.c	2016-07-21 13:26:42.408414261 -0400
+++ fig2dev-3.2.6-rc/fig2dev/dev/readpng.c	2016-07-21 13:34:28.790596818 -0400
@@ -18,6 +18,11 @@
 #include "colors.h"	/* rgb2luminance() */
 #include <png.h>
 
+#if defined(__WIN32__) && !defined(__CYGWIN__)
+  #include <string.h>
+  #define bcopy(a,b,c) memcpy(a,b,c)
+#endif
+
 /* return codes:  1 : success
 		  0 : invalid file
 */
