diff -ur fig2dev-3.2.8.orig/fig2dev/dev/genbitmaps.c fig2dev-3.2.8/fig2dev/dev/genbitmaps.c
--- fig2dev-3.2.8.orig/fig2dev/dev/genbitmaps.c	2020-12-24 07:58:17.831822681 -0500
+++ fig2dev-3.2.8/fig2dev/dev/genbitmaps.c	2020-12-24 07:59:41.971440443 -0500
@@ -493,8 +493,9 @@
 		err_msg("Cannot write command for conversion to bitmap");
 		BITMAP_EXIT_FAILURE;
 	}
-
+#if !defined (__WIN32__) || defined (__CYGWIN__)
 	(void) signal(SIGPIPE, bitmaps_broken_pipe);
+#endif
 	if ((tfp = popen(com, "w")) == NULL) {
 		fprintf(stderr, "fig2dev: Cannot open pipe to ghostscript\n");
 		fprintf(stderr, "command was: %s\n", com);
@@ -534,8 +535,9 @@
 
 	status = pclose(tfp);
 	tfp = NULL;	/* Otherwise main() tries to close tfp again */
+#if !defined (__WIN32__) || defined (__CYGWIN__)
 	(void) signal(SIGPIPE, SIG_DFL);
-
+#endif
 	if (status != 0) {
 		FILE	*f;
 
diff -ur fig2dev-3.2.8.orig/fig2dev/dev/genpdf.c fig2dev-3.2.8/fig2dev/dev/genpdf.c
--- fig2dev-3.2.8.orig/fig2dev/dev/genpdf.c	2020-12-24 07:58:17.831822681 -0500
+++ fig2dev-3.2.8/fig2dev/dev/genpdf.c	2020-12-24 08:01:13.368456502 -0500
@@ -111,7 +111,9 @@
 		exit(EXIT_FAILURE);
 	}
 
+#if !defined (__WIN32__) || defined (__CYGWIN__)
 	(void) signal(SIGPIPE, pdf_broken_pipe);
+#endif
 	if ((tfp = popen(com, "w")) == 0) {
 		err_msg("fig2dev: Cannot open pipe to ghostscript");
 		put_msg("Command was: %s", com);
@@ -145,7 +147,9 @@
 		err_msg("Error in ghostcript command,\n  %s", com);
 		status = -1;
 	} else {
+#if !defined (__WIN32__) || defined (__CYGWIN__)
 		(void)signal(SIGPIPE, SIG_DFL);
+#endif
 	}
 
 	if (com != com_buf)
