--- a/units.c.orig	2013-10-21 11:47:34 -0400
+++ b/units.c	2013-10-21 12:18:24 -0400
@@ -1166,23 +1166,23 @@
 int
 setenv(const char *name, const char *val, int overwrite)
 {
-  char *environ;
+  char *environ1;
 
   if (!overwrite && getenv(name) != NULL)
     return 0;
-  environ = (char *) malloc(strlen(name) + strlen(val) + 2);
-  if (!environ)
+  environ1 = (char *) malloc(strlen(name) + strlen(val) + 2);
+  if (!environ1)
     return 1;
-  strcpy(environ, name);
-  strcat(environ, "=");
-  strcat(environ, val);
+  strcpy(environ1, name);
+  strcat(environ1, "=");
+  strcat(environ1, val);
 
   /* putenv() doesn't copy its argument, so don't free environ */
 
 #if defined _WIN32 && defined _MSC_VER
-  return _putenv(environ);
+  return _putenv(environ1);
 #else
-  return putenv(environ);
+  return putenv(environ1);
 #endif
 }
 #endif
