diff -ur zeromq-4.0.3.orig/src/socket_base.cpp zeromq-4.0.3/src/socket_base.cpp
--- zeromq-4.0.3.orig/src/socket_base.cpp	2014-02-06 17:38:06.285815420 +0000
+++ zeromq-4.0.3/src/socket_base.cpp	2014-02-06 17:38:19.024897442 +0000
@@ -119,7 +119,7 @@
             errno = EINVAL;
             return NULL;
     }
-    if (s->mailbox.get_fd () == -1)
+    if (s->mailbox.get_fd () == retired_fd)
         return NULL;
 
     alloc_assert (s);
